### ISSUE TITLE
[Bug] Untitled Bug Issue

### ISSUE THREAD

**Issue by duypv** [AppFlowy-IO/AppFlowy-Cloud#973]
**Describe the bug**
Can't invite user to workspace

**To Reproduce**
1. Go to 'Invite another user to your workspace'
2. Type email
3. Click on 'Invite'

**Screenshots**
<img width="676" alt="image" src="https://github.com/user-attachments/assets/35ea671c-b8e0-498f-8a8b-b90d5ddbfd44">

Log gotrue:
{
  "component": "api",
  "error": "this token needs to have one of the following roles: service_role, supabase_admin",
  "level": "info",
  "method": "POST",
  "msg": "this token needs to have one of the following roles: service_role, supabase_admin",
  "path": "/admin/generate_link",
  "referer": "appflowy-flutter://",
  "remote_addr": "172.23.0.4",
  "request_id": "a73e40f6-1499-4497-8066-7a3b17870c7d",
  "time": "2024-11-08T05:42:51Z"
}

Log Applowy cloud
{
  "timestamp": "2024-11-08T04:56:44.118907Z",
  "level": "ERROR",
  "fields": {
    "error": "User not allowed"
  },
  "target": "appflowy_cloud::api::workspace",
  "filename": "src/api/workspace.rs",
  "line_number": 329,

**Comment by speed2exe** [comment]
@duypv You need be using admin email for the following env:
```
APPFLOWY_MAILER_SMTP_USERNAME=
APPFLOWY_MAILER_SMTP_PASSWORD=
```
which can be the same as:
```
GOTRUE_ADMIN_EMAIL=
GOTRUE_ADMIN_PASSWORD=
```

**Comment by duypv** [comment]
you mean it need to be same value?

Right now, magic link work but I can't invite other user.

**Comment by speed2exe** [comment]
@duypv Yes, you can use the same value.

**Comment by sylarevan** [comment]
Just saw this issue. I still have an error with email invitation, though I'm correctly setting `APPLOWY_MAIL_SMTP_*` values, see https://github.com/AppFlowy-IO/AppFlowy-Cloud/issues/553#issuecomment-2507986766

**Comment by nicksas** [comment]
Hi, have you found a solution?

**Comment by speed2exe** [comment]
@duypv Are you still facing this issue? The only way I can think of scenario in the above is where you have logged in as a non-admin user, but you somehow manage to get into the admin page `/web/admin/home`, for which even though you can see the functionalities, it will not allow you to perform any of the actions due to permission. It could be also you have logged in to another non-admin user from another tab, you can resolve this by logging out and logging in again.

**Comment by duypv** [comment]
yes, please note magic link work normal. But invite mail to workspace don't work. So I don't think SMTP config is reason.

**Comment by duypv** [comment]
on both: desktop app and admin ui. No email sent.

**Comment by speed2exe** [comment]
@duypv you actually dont need email for workspace invitation to work(it's for notification and convenience). you can have the invitee to login to your self host cloud `web/login`, go to "invite" section, you should see it in the pending invite section
